x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/routes/api/follower_index.ts

```diff
@@ -161,21 +185,10 @@ export const registerFollowerIndexRoutes = ({ router, __LEGACY }: RouteDependenc
    */
   router.put(
     {
-      path: `${API_BASE_PATH}/follower_indices/{id}`,
+      path: getEndpoint('followerIndex', 'edit').path,
```

Comment:
I'm not sure I see the benefit of the added `getEndpoint` machinery.

It seems like another thing we need to build out (`replacePlaceholders` 👀) and share properly across apps and without a clear benefit over statically declaring these values (at least the path pattern, not convinced about the method) and importing them from a common folder.

If we issue a request (in tests or in app code) against a server on a path that does not exist we should be met with a 404 and so the server is already the source of truth, so this does not seem like a major current pain point.

My understanding of the bug we had here was that we required a body but the logic inside the route handler accepted a body with all `undefined` values - combined with NP route validation the oversight snuck in because body without route validation is `{}` (i.e., an acceptable value!).

The best way, in my view, to cover this specific case would be the addition of an API integration test that issues a request against elastic with a value set and checking the response (like what you have added here `x-pack/test/api_integration/apis/management/cross_cluster_replication/follower_indices.js`)

The major take away for me, from this bug, besides the added tests is that we need to be more thorough in our route validations and explicitly testing sending unexpected values to routes.